tests/foreman/ui/test_computeresource_ec2.py

```diff
@@ -0,0 +1,382 @@
+
+        :expectedresults: An ec2 compute resource is not created
+
+        :Caseautomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_update_ec2_name(self):
+        """Update An ec2 compute resource name
+
+        :id: bfc43b84-518b-4ec9-a2ac-e77a1e7cdcb0
+
+        :setup: ec2 hostname and credentials.
+
+        :steps:
+
+            1. Create a compute resource of type ec2.
+            2. Provide a valid Access Key and Secret Key.
+            3. Provide a valid name to ec2 compute resource.
+            4. Test the connection using Load Regions and submit.
+            5. Update the name of the created CR with valid string.
+
+        :expectedresults: The ec2 compute resource is updated
+
+        :Caseautomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_update_ec2_organization(self):
+        """Update An ec2 compute resource organization
+
+        :id: ee286e04-0012-4573-86aa-8930efe89ec6
+
+        :setup: ec2 hostname and credentials.
+
+        :steps:
+
+            1. Create a compute resource of type ec2.
+            2. Provide a valid Access Key and Secret Key.
+            3. Provide a valid name to ec2 compute resource.
+            4. Test the connection using Load Regions and submit.
+            5. Create a new organization.
+            6. Add the CR to new organization.
+
+        :expectedresults: The ec2 compute resource is updated
+
+        :Caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_delete_ec2(self):
+        """Delete An ec2 compute resource
+
+        :id: fd1ba240-d47f-42c8-a1d1-146ab5fd2641
+
+        :setup: ec2 hostname and credentials.
+
+        :steps:
+
+            1. Create a compute resource of type ec2.
+            2. Provide a valid Access Key and Secret Key.
+            3. Provide a valid name to ec2 compute resource.
+            4. Test the connection using Load Regions and submit.
+            5. Delete the created compute resource.
+
+        :expectedresults: The compute resource is deleted
+
+        :Caseautomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier2
+    def test_positive_add_image_ec2_with_name(self):
+        """Add images to the ec2 compute resource with valid name.
+
+        :id: f70c9520-e8cb-46fb-99b6-045877c70170
+
+        :setup: ec2 hostname, credentials and images in ec2.
+
+        :steps:
+
+            1. Create a compute resource of type ec2.
+            2. Provide a valid Access Key and Secret Key.
+            3. Select the created ec2 CR and click images tab.
+            4. Select "New image" , provide it valid name and information.
+            5. Select the desired Amazon Machine Image (ami) to create image
```

Comment:
also, let's mention here that the user which we need to select needs to be "ec2-user" only, for any template commands to get executed on the host.